src/rgw/rgw_op.cc

```diff
@@ -2636,6 +2639,12 @@ void RGWStatAccount::execute()
       global_stats.buckets_count += m.size();
 
     }
+    if (!lastmarker) {
+	lderr(s->cct) << "ERROR: rgw_get_user_buckets, stasis at marker="
```

Comment:
apparent spelling error -- s/stasis/static/ ?

Reply:
stasis = an abnormal state in which the normal flow of a liquid is slowed or stopped; inactivity resulting from a static balance between opposing forces.  static = a crackling or hissing sound probably caused by electrical interference.  In this case, I choose stasis.

Reply:
Well if you're going to choose a definition of "static" decency demands you choose the definition most favorable to me, such as "lacking in movement, action, or change, especially in a way viewed as undesirable or uninteresting".

I'll leave the error message up to you. But given that ceph is used internationally by many people for whom English is not their first language, we should try to make the messages as widely understandable as possible. I'll just throw out "failure to make forward progress" in case it sounds good to you.

Reply:
I'm hoping no human ever sees this, so I'm leaving it as is.

Reply:
It looks like the function called above is `rgw_read_user_buckets` rather than `rgw_get_user_buckets`.

Reply:
and the warning message above on line 2610 also gets the function name wrong, so it'd be good to fix both